package-lock.json

```diff
@@ -12640,25 +12640,25 @@
       "dependencies": {
         "abbrev": {
           "version": "1.1.1",
-          "resolved": "",
+          "resolved": false,
```

Comment:
I've also been noticing npm sneaking these changes into the lockfile on `npm install`. It isn't overriding any urls, just `""` so it should be safe to proceed with. I tried finding out more about this but nothing worth noting. We can check it in the staging branch just to be sure, but I was still able to freshly `npm install` locally without any issues.